.github/workflows/build.yml

```
@@ -280,21 +276,13 @@ jobs:
         uses: jurplel/install-qt-action@b3ea5275e37b734d027040e2c7fe7a10ea2ef946
         with:
           version: '5.15.2'
-          arch: |-
-            ${{
-              fromJSON('
-                {
-                  "x86": "win32_msvc2019",
-                  "x64": "win64_msvc2019_64"
-                }
-              ')[matrix.arch]
-            }}
+          arch: "win64_msvc2019_64" # if fails, replace with x64
```

`x64` is the key, not the value.  Your logic is correct.

```suggestion
          arch: "win64_msvc2019_64"
```

---

done